core/templates/includes/titles_browse_ctrl.html

```diff
@@ -1,3 +1,6 @@
+<!-- I don't know if this is needed so I am not deleting it yet -kmd -->
```

Comment:
... and I deleted titles. We can bring it back if need be, I was trying to make templates less confusing.